source/index.html.md

```diff
@@ -448,17 +449,16 @@ curl \
 -H "X-API-Key: [partner-api-key]" \
 -X POST \
 -d '{
-{
   "external_round_id": "partner-round-id-1",
-  "game_id": "ePz8u6hbncrNzFRCqLbX8e",
+  "game_id": "ePz8u6hbncrNzFRCqLbX8e"
 }' https://ns-api-staging.betable.com/lfs/v100\
 /users/QEaakM0qumTJd5tj/rounds
 ```
 `POST /users/{user_id}/rounds`
 
 This method creates a round. A round is a container used for grouping together wagers and payouts that are associated to a user. Any combination of wagers and payouts can be contained within a round, as long as the round has not been closed.
 
-One user may share many wager and payout events as long as the game_id and user_id remain the same.
+One round may share many wager and payout events as long as the game_id and user_id remain the same.
```

Comment:
As Alan pointed out, this line is not clear. Did we mean a "round" here? Also, what does this even mean?

Reply:
A round can arbitrarily have 1 wager or payout event or many of them.  The only limiting factor is that a round must maintain the same game_id.  This wording can definitely be improved but I'm not sure how best to do that.

Reply:
Maybe: 

A round can consist of zero or more Wagers, and zero or more Payouts.  The choice of how many wagers and payouts to include in a round is up to the integration provider.  The only caveat is that all of the wagers and payouts within that round all belong to the same game as represented by the "game_name".